### PR TITLE
Use `files` property in package.json

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
     "url": "git://github.com/cheeriojs/dom-renderer.git"
   },
   "main": "./index.js",
+  "files": [
+    "index.js"
+  ],
```
Comment (davidchambers) [inline package.json]: Should we list `"package.json"` as well?

Comment (kevva) [inline package.json]: Nope.

Comment (davidchambers) [inline package.json]: Ah, I didn't know that. Thanks.

**Comment by davidchambers** [comment]
LGTM

**Pull request merged**